lib/resque/data_store.rb

```diff
@@ -97,7 +97,7 @@ def server_time
     end
 
     def redis_time_available?
-      @redis.time
+      @redis.time unless Resque.inline
```

Comment:
This returns null when inline is true, yes? Is it okay to return null instead of false? Otherwise looks good to me.